src/Compilers/CSharp/Portable/Symbols/MemberSymbolExtensions.cs

```diff
@@ -453,48 +453,6 @@ internal static ImmutableArray<Symbol> GetExplicitInterfaceImplementations(this
             }
         }
 
-        internal static TypeSymbol GetTypeOrReturnType(this Symbol member)
```

Comment:
I am referring to the fact that they operate on symbols that are not members as well as symbols that are members.

---
In reply to: [80849752](https://github.com/dotnet/roslyn/pull/13999#discussion_r80849752) [](ancestors = 80849752,80815690,80807640)